docs/rules/require-v-if-else-key.md

```diff
@@ -0,0 +1,52 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-v-if-else-key
+description: require `key` with `v-if/v-else-if/v-else` directives
+---
+# vue/require-v-if-else-key
+
+> require `key` with `v-if/v-else-if/v-else` directives
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule reports the elements which have `v-if`, `v-else-if`, and/or `v-else` and do not have a `v-bind:key` or `key`.
```

Comment:
An element can't have more than one of these directives. Also, it should be clarified that they have to have the same tag name.

```suggestion
This rule reports consecutive elements with the same tag name which all have `v-if`, `v-else-if`, or `v-else` and do not have a `v-bind:key` or `key`.
```